### PR TITLE
For commenting on 00-00-Preface.ipynb (part of review-865a5c2)

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

File: notebooks/00-00-Preface.ipynb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    "This astronomical content work was inspired by, and guided by, the excellent\n",
+    "resources below:\n",
+    "\n",
+    "+ [\"A User's Guide to CCD Reductions with IRAF\" (Massey 1997)](http://www.ifa.hawaii.edu/~meech/a399/handouts/ccduser3.pdf) is very thorough, but IRAF has become more\n",
+    "difficult to install over time and is no longer supported.\n",
+    "+ [\"A User's Guide to Stellar CCD Photometry with IRAF\" (Massey and Davis 1992)](https://www.mn.uio.no/astro/english/services/it/help/visualization/iraf/daophot2.pdf).\n",
+    "+ [The Handbook of Astronomical Image Processing](https://www.amazon.com/Handbook-Astronomical-Image-Processing/dp/0943396824) by Richard Berry and James Burnell. This\n",
+    "provides a very detailed overview of data reduction and photometry. One virtue\n",
+    "is its inclusion of *real* images with defects.\n",
+    "+ The [AAVSO CCD Obseving Manual]() provides a complete introduction to CCD data reduction and photometry.  \n",
```
Comment (saimn) [inline notebooks/00-00-Preface.ipynb]: Missing link, + typo: Observing

**Comment by mwcraig** [comment]
Closing, but noted in issue #234

File: notebooks/00-00-Preface.ipynb [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    "## Data files\n",
+    "\n",
+    "The list of the data files, and their approximate sizes, is below. You can\n",
+    "either download them one by one, or use the download helper included with these\n",
+    "notebooks.\n",
+    "\n",
+    "### Use this in a terminal to download the data\n",
+    "\n",
+    "```console\n",
+    "$ python download_data.py\n",
```
Comment (DaveStrickland) [inline notebooks/00-00-Preface.ipynb:132]: There doesn't seem to be a link for users to download download_data.py themselves, nor it is obviously in the github repo itself.

**Comment by JohnAstronomer** [comment]
The The Handbook of Astronomical Image Processing by Richard Berry and James Burnel is no longer available as Wilmen-Bell have ceased publishing. The only option is to buy secondhand copies

**Pull request closed without merging**